docs/CFDE-Internal-Training/cfdebot_website_editing.md

```diff
@@ -58,40 +59,49 @@ If you are satisfied with the edits, request reviewers to check, request changes
 
 When the PR has been approved, click "Merge pull request" and confirm merge. Be sure to delete your branch. Your edits should now be viewable on the public-facing website!
 
-## Specific steps for editing the CFDE documentation website
+## Specific steps for editing the CFDE documentation website <a name="published-docs"></a>
 
-The website created by the `published-documentation` repo pulls some docs that are in its repo AND some from a sub-module (`the-fair-cookbook`). There are two ways to make changes to this website.
+The website created by the `published-documentation` repo pulls some docs that are in its repo AND some from two sub-module repositories (`the-fair-cookbook` and `specifications-and-documentation`). There are two ways to make changes to this website that are detailed in the repo's [README](https://github.com/nih-cfde/published-documentation/blob/dev/README.md) documentation. The steps are summarized below:
 
 ### A) **To edit documents that are *in* the `published-documentation` repo**
 
-Follow the general steps above. Note that this repo currently labels branches as `master` instead of `stable`. This will be updated in the near future.
+Follow the general steps above, with the following additional steps:
 
-### B) **To edit documents that are in the sub-module `the-fair-cookbook` repo**
+- Push your changes to the `preview` branch first to check the rendered website
+- If the changes look as you expected, make a PR of your branch to `dev` and tag the admin team, who will check the changes and approve. Approved changes will periodically be promoted to the `stable` branch to be rendered on the public website.
```

Comment:
I think we need to add what they need to explicitly tag @admin otherwise they might ask themselves, how do we tag admin(assuming they are new to github)? Small point, but I think it would be helpful.